docs/src/main/tut/middleware.md

```diff
@@ -33,10 +33,10 @@ the wrapped service like this.
 
 ```tut:book
 def myMiddle(service: HttpService, header: Header): HttpService = Service.lift { req =>
-  service(req).map {resp =>
-    if (resp.status.isSuccess)
+  service(req) map {
```

Comment:
We're preferring infix syntax now? This is really going to mess me up between work where we're trying to get rid of it and here. :smile: